src/styles/_card.scss

```
@@ -0,0 +1,100 @@
+$color-main: #060b35;
+$bg-color-main: #f3f3f3;
+$color-secondary: #616070;
+$color-button: #00acdc;
+
+.main {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  width: 944px;
+  margin: 0 auto;
+  padding-top: 50px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  border: 1px solid $bg-color-main;
+  margin-bottom: 46px;
+
+  &__image {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 134px;
+    margin-bottom: 40px;
+    padding-right: 3px;
+    padding-left: 3px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-main;
+    margin-bottom: 4px;
+  }
+
+  &__product-code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    margin-bottom: 16px;
+    color: $color-secondary;
+  }
+
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+
+    &-reviews {
+      font-size: 10px;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+
+    &-string {
+      font-size: 12px;
+      line-height: 18px;
+      color: $color-secondary;
+    }
+
+    &-numbers {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+    }
+  }
+
+  &__button-buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
+    color: #fff;
+    background-color: $color-button;
+    border: solid 1px $color-button;
+    border-radius: 5px;
+    text-decoration: none;
+    text-transform: uppercase;
+    font-size: 14px;
+    font-weight: 700;
+
+    &:hover {
+      color: $color-button;
+      background-color: #fff;
```

You created variables for colors, but not for all 🥺 
```suggestion
      background-color: $c-white;
```